lib/pages/setup/route_list.dart

```diff
@@ -53,89 +54,167 @@ class _RouteListState extends State<RouteList> {
 
     setState(() {
       widget.routeParams.routes = routes;
-      print('## ${routes.length} routes found');
-
-      widget.routeParams.routes.forEach((r) => routeList.add(RouteListEntry(
-            r.title,
-            r.date,
-            r.totalLength,
-          )));
-      
+      widget.routeParams.routes.forEach((r) => _routeList.add(RouteListEntry(r)));
+      if(routes.length > 0) _title = routes[0].title;
       this._routesCalculated = true;
     });
 
   }
 
+  // TODO add localization or remove if not needed
+  headerText() {
+    String paramTitles = 'Start: ';
```

Comment:
This is really bad practice. Because on different devices the text is displayed differently with this approach.
A better way would be creating a row and add each text as a separate widget e.g.:

```
Row:  |Start:                     |Musterstraße 3, Berlin |
Row:  |Distance                   |5 km / 60 min          |
Row:  |Altitude difference        |n.a.                   |
```

Alternatively there are more widgets to style this for example a table.

Reply:
Newly styled within table